media-plugins/distrho-ports/distrho-ports-99999999.ebuild

```diff
@@ -1,22 +1,14 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-inherit meson
+inherit meson git-r3
 
 DESCRIPTION="Linux ports of Distrho plugins"
 HOMEPAGE="https://github.com/DISTRHO/DISTRHO-Ports"
 EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports"
-if [[ ${PV} == *9999 ]]; then
-	inherit git-r3
-	KEYWORDS=""
-else
-	MY_PN="${PV:0:4}-${PV:4:2}-${PV:6:2}"
-	SRC_URI="https://github.com/DISTRHO/DISTRHO-Ports/archive/${MY_PN}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64"
-	S="${WORKDIR}/DISTRHO-Ports-${MY_PN}"
-fi
+KEYWORDS=""
```

Comment:
Is there a reason to remove the `if` block? Keeping the `if` block would allow us to only maintain the live ebuild and copy it over to new versioned ebuilds upon new releases.